### PR TITLE
Figshare data

### PR DESCRIPTION
**Pull request by johnne**
In this PR we remove the dependency on the `sra-tools` package for downloading fastq files. This is because the package is having network errors in some cases when using `fastq-dump`, see for instance:

- https://github.com/ncbi/sra-tools/issues/790
- https://github.com/ncbi/sra-tools/issues/777
- https://github.com/ncbi/sra-tools/issues/788

In order to remove the dependency I uploaded subsampled fastq files to the SciLifeLab Data Repository [here](https://figshare.scilifelab.se/articles/educational_resource/MRSA_case_study_example_data/22246417). These files were created by running `seqtk sample <sample_id>.fastq.gz 100000`.

To make use of these files in the course Snakemake workflow the `get_SRA_by_accesion` rule was updated to:

```python
def get_sample_url(wildcards):
    return config["sample_urls"][wildcards.sample_id]

rule get_SRA_by_accession:
    """
    Retrieve a single-read FASTQ file
    """
    output:
        "data/raw_internal/{sample_id, \w+}.fastq.gz"
    params:
        url = get_sample_url
    shell:
        """
        curl -L {params.url} | seqtk sample - 20000 | gzip -c > {output[0]}
        """
```

This meant introducing the concept of input functions quite early in the tutorial. I think the best thing is actually to introduce input functions already at the start of the tutorial. This change has cascading effects throughout the tutorial.

### CODE REVIEW

File: pages/conda/conda-2-the-basics.md
Comment (fasterius) [inline pages/conda/conda-2-the-basics.md:1]: Instances of `conda` needs to be changed to `mamba` where applicable, to be in line with #209.

**Review by fasterius — CHANGES_REQUESTED** [review]
Great work! Some instances of `conda` needs to be changed to `mamba` to adhere to the new changes introduced in #209. The Nextflow material will also need to be updated accordingly.

**Comment by johnne** [comment]
@fasterius, thanks! Yes I wanted to wait with changing conda in this branch before that PR was merged. Will start updating. Do you want to handle the nextflow changes? I was thinking maybe you wanted to do that change together with the rest of the Nextflow update, or I could change it first?

**Comment by fasterius** [comment]
Sure, I can do the Nextflow things! I will try to do it this week or next.

**Pull request merged**